lib/streamlit/elements/map.py

```diff
@@ -186,7 +194,7 @@ def to_deckgl_json(data: Data, zoom: Optional[int]) -> str:
     #  the empty attribute. This is either a bug, or the documented data type
     #  is too broad. One or the other should be addressed, and the ignore
     #  statement removed.
-    if data[lon].isnull().values.any() or data[lat].isnull().values.any():  # type: ignore[index]
+    if data[lon].isnull().values.any() or data[lat].isnull().values.any():
```

Comment:
nit: If we can remove the ignore statement here, we can also remove the comment from harahu.

Reply:
It's removed 👍